packages/augur-ui/src/modules/market-charts/components/market-outcome--header-orders/market-outcome--header-orders.tsx

```diff
@@ -1,12 +1,82 @@
 import React from "react";
 import PropTypes from "prop-types";
 import classNames from "classnames";
-
-import { collapseIcon, expandIcon } from "modules/common/components/icons";
-
 import StylesHeader from "modules/market/components/market-outcomes-list/market-outcomes-list.styles.less";
 import Styles from "modules/market-charts/components/market-outcome--header-orders/market-outcome--header-orders.styles.less";
 
+const collapseIcon = (
```

Comment:
should these icons go in icons.tsx?

Reply:
they are only used once, so i moved them to where they are used simply because eventually we should replace them with icons that are more common/reused. I didn't want to pollute the icons file with more one off icons, there are a few in there now but they are really specific to like menu icons which matt said was going away anyway. I added TODOs for those to remove them when we get the redesigned menu